minetest/minetest
# DevTest: Fix armorball sprite

The sprite of the DevTest armorball was broken (incorrect graphics), so this PR fixes it.

How to test:
* Start DevTest
* Spawn `testentities:armorball`
* Enter Debug Mode
* Rightclick the armorball to cycle through the armor modes
* Check if the sprite graphics look correct
